src/device.c

```diff
@@ -40,6 +40,8 @@
 #include "poll.h"
```

Comment:
Please, remove the lines which includes "poll.h" in the commit 46497f00c9d8faa094af629c5fd731befc153185.